pkg/products/observability/reconciler.go

```diff
@@ -165,7 +200,167 @@ func (r *Reconciler) Reconcile(ctx context.Context, installation *integreatlyv1a
 	return integreatlyv1alpha1.PhaseCompleted, nil
 }
 
-func (r *Reconciler) reconcileSecrets(_ context.Context, _ k8sclient.Client, _ *integreatlyv1alpha1.RHMI) (integreatlyv1alpha1.StatusPhase, error) {
+func (r *Reconciler) reconcileAlertManagerSecrets(ctx context.Context, serverClient k8sclient.Client) (integreatlyv1alpha1.StatusPhase, error) {
```

Comment:
is this function the same as the one used for managing the resources for AMO?  is it possible to put the shared logic somewhere?

Reply:
As AMO won't be around after we migrate to OO taught temporary duplication of code would be ok. But AMO will still be around after for rhmi so will see about moving it.